### ISSUE TITLE
Some improvement

### ISSUE THREAD

**Issue by franco999** [bwr57/CustomizePublicTransportStop#2]
1. The tag bus=yes is added in the relation and does not correspond there.
2. You could add the tag value *=no, (shelter=no, bench=no, etc)
3. You could add more options tags as "bin, level, layer, unsigned, ref, tactile_paving, passenger_information_display"

**Comment by bwr57** [comment]
First point fixed

**Comment by franco999** [comment]
@bwr57 Will this repository be updated with some of the proposed fixes? It is necessary, since both I and many others, use this plugin that facilitates editing and life.

**Comment by AgusQui** [comment]
instead of passenger_information_display i prefer departures_board=yes/realtime/timetable/delay/no

**Comment by franco999** [comment]
@AgusQui You're right, I like that one better. But I think this project is abandoned, it's not even in the JOSM plugin list anymore.

**Comment by ialokim** [comment]
> it's not even in the JOSM plugin list anymore

And it stopped working some JOSM updates before. I've done some minor changes to allow further using on [my fork](/ialokim/CustomizePublicTransportStop/). With some free time, I could eventually work on adding more functionalities in some future, too.

**Comment by floscher** [comment]
Everything from this JOSM plugin is now part of the [`pt_assistant` plugin](https://github.com/JOSM/pt_assistant) starting with [release `2.1.10`](https://github.com/JOSM/pt_assistant/releases/tag/v2.1.10).

That plugin is currently more actively maintained than this one, feel free to open issues against the `pt_assistant` plugin: https://josm.openstreetmap.de/newticket?component=Plugin%20pt_assistant

**Comment by bwr57** [comment]
> t I think this project is abandoned, it's not even in the JOSM plugin list anymore.

I checked JOSM plugin list. All is OK. The version in the svn repository have a fixes for the last version of the JOSM

**Comment by floscher** [comment]
Hi @bwr57, sorry that I didn't contact you earlier! Based on that the latest commit in this repository is now over two years old, I assumed this project was kind of abandoned. Sincere apologies for simply assuming and not asking.

Well, that being said, the job of updating the plugin to newer JOSM versions was since August 2016 done by the core maintainers @don-vip and @stoecker (not counting translation updates). The idea of merging some of the plugins that do a similar job is an effort in reducing the maintenance burden (esp. for the core developers) of currently 100+ plugins.
Just to explain: The merge that I perfomed with `CustomizePublicTransportPlugin` and `pt_assistant` hasn't really changed anything of your code, it's now just as if you previously had both plugins installed at once.

It would be great if we could in some way join forces with these projects instead of maintaining two so similar projects separately. I'd be willing to discuss this of course, if you would like to solve this differently. E.g. at [josmTicket#17005](https://josm.openstreetmap.de/ticket/17005), originally opened to remove this plugin from the SVN repository (which would then also remove the plugin from the JOSM plugin list), but now let's first decide on how to continue.
